bundle/filefs.go

```diff
@@ -23,16 +23,26 @@ type dirLoaderFS struct {
 	files      []string
 	idx        int
 	filter     filter.LoaderFilter
+	root       string
+	pathFormat PathFormat
 }
 
 // NewFSLoader returns a basic DirectoryLoader implementation
 // that will load files from a fs.FS interface
-func NewFSLoader(filesystem fs.FS) (DirectoryLoader, error) {
+func NewFSLoader(filesystem fs.FS) DirectoryLoader {
```

Comment:
Nit: Let's not change the interface here ie  keep `(DirectoryLoader, error)` as-is.